### PR TITLE
Remove deprecated metrics-tracker

### PR DESCRIPTION
**Pull request by rhagarty**
Remove:
Badge at top of README
Privacy Notice from Readme
Disabling Deployment Tracking from Readme
use of metric tracker in app:
i.e. require('metrics-tracker-client').track();
package.json for metrics-tracker
package-lock.json stuff

Change D2B to:
https://bluemix.net/deploy/button.png

### CODE REVIEW

File: test/__snapshots__/searchField.test.js.snap [hunk 1]
```diff
@@ -45,7 +45,9 @@ exports[`renders correctly 1`] = `
         <input
           checked={false}
           className="hidden"
+          id={undefined}
```
Comment (scottdangelo) [inline test/__snapshots__/searchField.test.js.snap]: Why are things removed from this file?

File: package-lock.json [hunk 2]
```diff
@@ -3270,15 +3270,6 @@
         "lodash.get": "4.4.2"
       }
     },
-    "cwd": {
```
Comment (scottdangelo) [inline package-lock.json]: Why are so many things removed from package-lock.json, other than metrics-tracker and it's dependencies?

Comment (rhagarty) [reply package-lock.json]: Again, not sure why the big changes. Assuming newer packages available since last build.

**Comment by rhagarty** [comment]
This can be merged. I tested with these changes.

**Pull request merged**